### PR TITLE
remove orphaned cosaves on startup

### PR DESCRIPTION
**Pull request by Kole6738**
this is really ugly and to be honest may not even compile - I couldn't get the base engine fixes to compile on my machine so this hasn't _actually_ been _tested_ or anything. It _should_ work but no guarantees and I'd keep it disabled by default until the more adventurous folk have found all the bugs. Anyway, this shouldn't ever need to be ported or anything if a new version of skyrim comes out since it's all handled in the engine fix plugin itself. Anyway, have fun fixing up my spaghetti code!

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -11,13 +11,92 @@
...
+			), sksecosaveVector.end()
+		);
+		for (auto &value : sksecosaveVector) {
+			if (remove(a=value) != 0)
+				_MESSAGE("something went wrong while removing orphans, bailing");
+			else
+				_MESSAGE("orhpan file \"" + value + "\" killed");
+		}
+	}
+	CloseHandle(hFind);
```
Comment (Kole6738) [inline main.cpp]: we're tidy kiwis around here

**Comment by aers** [comment]
Hey thanks I'll take a look at this tonight when I get the chance.

I'm gonna assume you can't get it to compile due to missing TBB.

**Comment by Kole6738** [comment]
Feel free to rename and refactor - I doubt the variable names make much sense to anyone but myself. Thanks for taking a look at it. I was tired of having to run the VBS someone made that cleans the folder, and so I decided that I would make my own save folder cleaner.

**Comment by aers** [comment]
Hey,

Ended up rewriting a bunch so I didn't bother to merge the PR. Thanks for the help :)

**Pull request closed without merging**